src/CrashlogFile.ts

```diff
@@ -79,6 +89,106 @@ export class CrashlogFile {
             const line = lines[i];
             this.findPkgPaths(line, i);
         }
+
+        this.parseCrashes();
+    }
+
+    private parseCrashes() {
+        this.crashes = [];
+        let contents = this.fileContents;
+
+        // TODO: redundant code from StandardReporter, how to refactor?
+        // walk the references backwards and replace the file contents
+        for (const ref of [...this.references].reverse()) {
+            //if the reference has a mapped source location
+            if (ref.srcLocation) {
+                contents =
+                    contents.substring(0, ref.offset) +
+                    standardizePath`${ref.srcLocation.path}` +
+                    `(${ref.srcLocation.line + 1})` +
+                    contents.substring(ref.offset + ref.length);
+            }
+        }
+
+        let crashReportBlocks = contents?.split(/\s*___+\s*/) ?? [];
+
+        // Filter out crash reports without stack trace
+        crashReportBlocks = crashReportBlocks.filter(block => !block.includes('StackTrace missing'));
+
+        crashReportBlocks.forEach(crashReportBlock => {
+            const crashReportBlockSections: Array<{ sectionType: CrashReportSectionType; lines: string[] }> = [];
+            const blockLines = crashReportBlock.split(/\r?\n+/).map(x => x.trim());
+
+            // Separate the block in three sections:
+            // Hardware Platform, Application Version, and Stack Trace
+            let currentSection: CrashReportSectionType | undefined;
+            let foundSectionHeader = false;
+            for (const line of blockLines) {
+                if (/\s*count\s+Hardware Platform/.test(line)) {
+                    currentSection = CrashReportSectionType.HardwarePlatform;
+                    foundSectionHeader = true;
+                } else if (/\s*count\s+Application Version/.test(line)) {
+                    currentSection = CrashReportSectionType.ApplicationVersion;
+                    foundSectionHeader = true;
+                } else if (/\s*Stack Trace/.test(line)) {
+                    currentSection = CrashReportSectionType.StackTrace;
+                    foundSectionHeader = true;
+                } else {
+                    if (foundSectionHeader && currentSection && !/\s*---+\s*/.exec(line)) {
+                        // TODO: What's this?
+                        // eslint-disable-next-line @typescript-eslint/no-loop-func
```

Comment:
This is an annoying rule. Go ahead and disable it in `eslintrc.json` and then remove these comments.

Reply:
Done on the latest commit.